internal/app/secretless/providers/providers.go

```diff
@@ -19,4 +20,5 @@ var ProviderFactories = map[string]func(plugin_v1.ProviderOptions) plugin_v1.Pro
 	"keychain": keychainProvider.ProviderFactory,
 	"literal":  literalProvider.ProviderFactory,
 	"vault":    vaultProvider.ProviderFactory,
+	"kubernetes-secrets":  kubernetesProvider.ProviderFactory,
```

Comment:
Nees alphabetic sort too

Reply:
Maybe the name could be a bit shoter? `k8s/secret` or something similar?

Reply:
addressed the first point. i'm going for kubernetes for the sake of being abundantly clear